10_deploy_rook.sh

```diff
@@ -7,27 +7,42 @@ source common.sh
 figlet "Deploying rook" | lolcat
 eval "$(go env)"
 
+export MIXINPATH="$GOPATH/src/github.com/ceph/ceph-mixins"
 export ROOKPATH="$GOPATH/src/github.com/rook/rook"
 cd $ROOKPATH/cluster/examples/kubernetes/ceph
-oc create -f common.yaml
+
+sed 's/name: rook-ceph$/name: openshift-storage/' common.yaml > common-modified.yaml
+sed -i 's/namespace: rook-ceph/namespace: openshift-storage/' common-modified.yaml
+oc create -f common-modified.yaml
+oc label namespace openshift-storage  "openshift.io/cluster-monitoring=true"
+oc policy add-role-to-user view system:serviceaccount:openshift-monitoring:prometheus-k8s -n openshift-storage
+
 sed '/FLEXVOLUME_DIR_PATH/!b;n;c\          value: "\/etc/kubernetes\/kubelet-plugins\/volume\/exec"' operator-openshift.yaml > operator-openshift-modified.yaml
```

Comment:
no longer needed as it was merged upstream

Reply:
ack.